src/pages/signup-affiliates/components/_account-type.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import styled from 'styled-components'
+import IndividualIcon from 'images/svg/signup-affiliates/individual 2.svg'
+import BusinessIcon from 'images/svg/signup-affiliates/company 2.svg'
+import Card, { CardProps } from 'components/custom/_card'
+import { localize, Localize } from 'components/localization'
+import { Header } from 'components/elements'
+
+const AccountType = () => {
+    const MainWrapper = styled.div`
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+    `
+    const card: CardProps[] = [
```

Comment:
it's a collection and thus the variable name should be plural.

Reply:
ok